Ncm 2025 flame speed sensitivity + plasma

BangShiuh opened the pull request:
I have updated the script and ran with cantera dev. version (3.2).

BangShiuh committed:
add flame speed sensitivity ran with cantera dev and passed

BangShiuh committed:
add plasma reaction jupyter py file ran with dev. (3.2)

ischoegl commented on ncm-2025/completed-notebooks/06_flame_speed_sensitivity_completed.py:
```diff
@@ -0,0 +1,170 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
```
```suggestion
```
Should be deleted as it is only meaningful to Jupyter lab. In VS Code, it produces irrelevant output.

ischoegl commented on ncm-2025/completed-notebooks/06_flame_speed_sensitivity_completed.py:
```diff
@@ -0,0 +1,170 @@
+#     name: python3
+# ---
+
+# # Flame Speed
```
```suggestion
# ## Flame Speed
```
Indent to a minimum heading level 2 as otherwise it will not render properly in VS Code.

ischoegl commented on ncm-2025/completed-notebooks/06_flame_speed_sensitivity_completed.py:
```diff
@@ -0,0 +1,170 @@
+#
+# The figure below illustrates the setup, in a flame-fixed co-ordinate system. The reactants enter with density $\rho_{u}$, temperature $T_{u}$ and speed $S_{u}$. The products exit the flame at speed $S_{b}$, density $\rho_{b}$ and temperature $T_{b}$.
+
+# <img src="../images/flameSpeed.png" alt="Freely Propagating Flame" style="width: 300px;"/>
```
Image should be downloaded from the web, and set a white background with border, see examples in other notebooks.

ischoegl commented on ncm-2025/completed-notebooks/plasma_reaction.py:
See comments in other script.

ischoegl reviewed (commented):
I have some minor comments that mainly ensure that scripts are usable from multiple IDEs. This can be done now or later ...

ischoegl commented:
Merging as there won't be time for feedback.

ischoegl commented:
@BangShiuh ... I fixed the few things I found after merging.

BangShiuh commented:
Thanks!

On Fri, Mar 14, 2025 at 5:15 PM Ingmar Schoegl ***@***.***>
wrote:

> @BangShiuh <https://github.com/BangShiuh> ... I fixed the few things I
> found after merging.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cantera/workshop-materials/pull/17#issuecomment-2725791875>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEZZJBTHDROJUZNRKN4DEOL2UNBI7AVCNFSM6AAAAABY7C4L62VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRVG44TCOBXGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: ischoegl]*ischoegl* left a comment (Cantera/workshop-materials#17)
> <https://github.com/Cantera/workshop-materials/pull/17#issuecomment-2725791875>
>
> @BangShiuh <https://github.com/BangShiuh> ... I fixed the few things I
> found after merging.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Cantera/workshop-materials/pull/17#issuecomment-2725791875>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEZZJBTHDROJUZNRKN4DEOL2UNBI7AVCNFSM6AAAAABY7C4L62VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMRVG44TCOBXGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
